common/app/experiments/Experiments.scala

```diff
@@ -13,6 +13,7 @@ object ActiveExperiments extends ExperimentsDefinition {
     DCRBubble,
     NGInteractiveDCR,
     NewsletterEmbedDesign,
+    UseAusCmp
```

Comment:
Can we have a trailing `,` comma?

Reply:
not in scala

Reply:
How comes the line above had one?

Reply:
oooh that is an EXCELLENT point!

Reply:
an you know what, as i replied i thought "i sound far more confident here than i should/am" 🤦

Reply:
And this seems to have been the reason for the build failure, too?